README.md

```diff
@@ -11,6 +11,12 @@ to provide GPU support, but this also means that NEML2 models have all the featu
 
 NEML2 is provided as open source software under a MIT [license](https://raw.githubusercontent.com/reverendbedford/neml2/main/LICENSE).
 
+### Build and installation
+
+Building should be as easy as cloning the repository, configuring with CMake, building with `make`, and testing with `make test`.
+
+By default NEML2 will download the current CPU-only version of torch.  To instead use a system torch set CMake options `-DSYSTEM_TORCH=ON -DTORCH_DIR=/path/to/your/torch`.  If you use the default build you will get a CPU-only version of torch and performance will suffer compared to a CUDA version.
+
```

Comment:
If you make the changes I suggested above, don't forget to update this sentence.